src/Apps/Legion/Hooks/useTypes.tsx

```diff
@@ -75,23 +75,32 @@ export const useTypes = () => {
         [wizardDataState]
     );
 
+    // Single item
+    const getTypeById = (typeId: string): IViewType => {
```

Comment:
let's make this a useCallback so that it doesn't recreate the object down below and bust memoization